File: dddnetcore/Domain/OperationTypes/CreatingOperationTypeDto.cs
using System.Collections.Generic;

namespace DDDSample1.Domain.OperationTypes{
    
    public class CreatingOperationTypeDto{
        public string Name { get; set; }
        public int EstimatedDuration  { get; set; }
        
        public int AnesthesiaTime { get; set; }

        public int CleaningTime { get; set; }
    
        public int SurgeryTime { get; set; }
        public List<CreatingStaffSpecializationDto> StaffSpecializations { get; set; }
    }
}

File: frontend/src/app/domain/CreatingOperationTypeDto.ts
import { CreatingStaffSpecializationDto } from "./creatingStaffSpecializationDto";

export interface CreatingOperationTypeDto {
    name: string;
    estimatedDuration: number;
    surgeryTime: number;
    anesthesiaTime: number;
    cleaningTime: number;
    staffSpecializations: CreatingStaffSpecializationDto[];
}